devito/operator/operator.py

```diff
@@ -751,8 +751,12 @@ def _emit_apply_profiling(self, args):
 
             v = summary.globals.get('fdlike')
             if v is not None:
-                perf("%s* Achieved %.2f FD-GPts/s" % (indent, v.gpointss))
-
+                if v.gflopss is not None:
```

Comment:
remind me when it could be None ?

Reply:
It happened when test_autotuning was running.
I will check again.

Reply:
OK, I see.

Can we show the GFlops/s first and the FD-GPts/s later? to make it homogeneous with the other prints

Reply:
ok